docs-source/content/samples/simple/azure-kubernetes-service/troubleshooting.md

```diff
@@ -0,0 +1,334 @@
+---
+title: "Troubleshooting"
+date: 2020-11-24T18:22:31-05:00
+weight: 3
+description: "Troubleshooting."
+---
+
+
+- [Access Administration Console](#fail-to-access-administration-console): possible causes for Administration Console inaccessibility.
+- [Domain debugging](#domain-debugging)
+- [Pod Error](#get-pod-error-details): how to get details of the pod error.
+- [WebLogic Image Tool failure](#weblogic-image-tool-failure)
+- [WebLogic Kubernetes Operator installation failure](#weblogic-kubernetes-operator-installation-failure)   
+   - [System pods are pending](#the-aks-cluster-system-pods-are-pending)
+   - [WebLogic Kubernetes Operator ErrImagePull](#weblogic-kubernetes-operator-errimagepull)
+- [WSL2 bad timestamp](#wsl2-bad-timestamp)
+- [Cannot attach ACR due to not being Owner of subscription](#cannot-attach-acr-due-to-not-being-owner-of-subscription)
+- [Virtual Machine size is not supported](#virtual-machine-size-is-not-supported)
+
+#### Get pod error details
+
+You may get the following message while creating the WebLogic domain: "the job status is not Completed!"
```

Comment:
"the job status is not Completed!" -> "`the job status is not Completed!`" (code font)